### PR TITLE
Fixed the docstring of RandomResizedCrop

### LINKED ISSUE #2357: RandomResizedCrop aspect_ratio is wrong
https://github.com/pytorch/vision/blob/1a04d3c265679e1a508e7cd627006aaa9ef1ccfb/torchvision/transforms/transforms.py#L699-L714

Doc says that `ratio` is the range of the ratio of the ***original aspect ratio***, yet the sampling code implies that the range is just directly the crop aspect ratio.

### PR DESCRIPTION
**Pull request by frgfm**
Hello there :wave: 

As I was myself a bit confused about the meaning of `RandomResizedCrop` arguments, I made a small edit suggestion to close #2357.

Any feedback is welcome!

### CODE REVIEW

File: torchvision/transforms/transforms.py [hunk 1]
```diff
@@ -739,7 +739,7 @@ class RandomResizedCrop(torch.nn.Module):
             int instead of sequence like (h, w), a square output size ``(size, size)`` is
             made. If provided a tuple or list of length 1, it will be interpreted as (size[0], size[0]).
         scale (tuple of float): range of size of the origin size cropped
```
Comment (vfdev-5) [inline torchvision/transforms/transforms.py:741]: Could you please also rewrod the docstring for scale.

File: torchvision/transforms/transforms.py [hunk 2]
```diff
@@ -739,7 +739,7 @@ class RandomResizedCrop(torch.nn.Module):
...
-        ratio (tuple of float): range of aspect ratio of the origin aspect ratio cropped.
+        ratio (tuple of float): range of the crop's aspect ratio before resizing.
```
Comment (vfdev-5) [inline torchvision/transforms/transforms.py:742]: How about that
```
ratio (tuple of float): aspect ratio ranges of the cropped image before resizing.
```

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR @frgfm 
I left few comments

File: torchvision/transforms/transforms.py (hunk 2 above)
Comment (frgfm) [reply torchvision/transforms/transforms.py]: Sounds good!

File: torchvision/transforms/transforms.py (hunk 1 above)
Comment (frgfm) [reply torchvision/transforms/transforms.py]: done!

**Review by fmassa — APPROVED** [review]

**Review by vfdev-5 — APPROVED** [review]
LGTM! Thanks @frgfm !

**Pull request merged**